### ISSUE TITLE
Possibly incorrect info for Global Functions in manual

### ISSUE THREAD

**Issue by acgetchell** [CGAL/cgal#378]
http://doc.cgal.org/latest/Kernel_23/group__barycenter__grp.html#gadd9f34ad5ecb5573a2fab5be63e8142a

states that barycenter() can be accessed using:

````c++
#include <CGAL/global_functions.h>
````
However, this doesn't resolve:

In file included from ../unittests/S3ErgodicMovesTest.cpp:19:
../src/S3ErgodicMoves.h:34:10: fatal error: 'CGAL/global_functions.h' file not found
#include <CGAL/global_functions.h>
         ^
1 error generated.

````c++
#include <CGAL/barycenter.h>
````
Works.

**Comment by lrineau** [comment]
@palliez Can you have a look at this issue? `<CGAL/barycenter.h>` is from the PCA package, and the globsl functions should be in kernels.

**Comment by sloriot** [comment]
This has nothing to do with PCA, the include file use in the doc is wrong.

**Comment by sloriot** [comment]
Fixed by 103687f
